### DISCUSSION TITLE
Yellowish border appears on map focus (mobile)

### DISCUSSION THREAD

**Discussion by Mis1eader-dev** [maplibre/maplibre-gl-js#5069] [Q&A]
**maplibre-gl-js version**: v4.7.0

**browser**: Google Chrome

### Steps to Trigger Behavior

 1. Create a map that spans the entire screen
 2. In the CSS give it an offset so that the edges could be visible, this can be done with `position: relative; left: -4px; top: -4px;`
 3. In devtools toggle device emulation.
 4. Give the map focus either programmatically, or through keyboard focus (tab).

### Link to Demonstration
![image](https://github.com/user-attachments/assets/d7ce84bd-d9f7-4217-a004-90c7c9ffb77e)

### Expected Behavior
Maps should not have any borders or outlines unless explicitly stated by the programmer.

### Actual Behavior
Yellow border appears on map focus. Only workaround is to span the map sufficient pixels outside of bounds so it no longer shows on mobile:
```css
position: relative;
left: -2px;
top: -2px;
width: calc(100% + 2px * 2);
height: calc(100% + 2px * 2);
```

**Comment by HarelM** [comment]
The template is here for a reason, please create a minimal reproduction. 
Also, please check that this is not some accessibility definition on your browser.

**Comment by Mis1eader-dev** [comment]
@HarelM this is the minimal reproduceable example:
```html
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width,initial-scale=1.0">
		<title>🗺️</title>
		<link href="https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css" rel="stylesheet">
		<style>
			#map
			{
				position: absolute;
				left: -20px;
				top: -20px;
				width: 100%;
				height: 100%;
			}
		</style>
	</head>
	<body>
		<div id="map"></div>
		<script src="https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js"></script>
		<script>
			new maplibregl.Map({
				container: "map",
				style: {
					version: 8,
					sources: {
						"osm": {
							type: "raster",
							tiles: ["https://tile.openstreetmap.org/{z}/{x}/{y}.png"],
						}
					},
					layers: [
						{
							id: "osm",
							type: "raster",
							source: "osm",
						},
					],
				},
			});
		</script>
	</body>
</html>
```
---
Run it using VSCode Live Server or any other backend, open Chrome devtools, toggle device emulation, select a mobile device type, then press tab to shift keyboard focus on the map, and the yellow border will appear:
![image](https://github.com/user-attachments/assets/db9711b8-61a2-4df2-ad0c-d71fbca202da)

**Comment by HarelM** [comment]
I don't see any problem with this - this is meant for accessibility as far as I understand to let the users know what they currently focusing on.
I've pressed a few more times "tab" to focus on the attribution, which also get a yellow border...
![image](https://github.com/user-attachments/assets/1e294efd-9b17-4b14-9548-1ea3f729df04)

JSBIN:
https://jsbin.com/rijunoxiro/1/edit?html,output

**Comment by Mis1eader-dev** [comment]
The problem is if the map is meant to span the entire screen it will feel bugged to users when around the map there is a yellow border at all times

**Comment by HarelM** [comment]
If it you want to avoid this border (although I don't think you should due to accessibility requirement), you can create a man that is a bit bigger than the screen by a few pixels...

**Comment by Mis1eader-dev** [comment]
@HarelM thank you for the suggestion, I have already been using that to avoid the yellow borders on mobile. Is there no way to disable this feature? I'd imagine this is added intentionally as normal focusable elements usually have the `outline` CSS property to customize the border or look.
